check/tests/completion.rs

```diff
@@ -119,8 +119,8 @@ fn binop() {
 
     let (mut expr, result) = support::typecheck_expr(r#"
 let (++) l r =
-    l #Int+ 1
-    r #Float+ 1.0
+    prim.add_Int l 1
```

Comment:
Either add dummy functions with the correct type signature and call them
```
let add_Int x : Int -> Int -> Int = add_Int x
```
or add the basic functions we need to the `MockEnv`

or add a way to import modules in just the typechecker. That would be nice but the macro system is currently tied into the vm module so probably not feasible atm.